Sources/StreamChatUI/Composer/ComposerVC.swift

```diff
@@ -254,25 +274,28 @@ open class _ComposerVC<ExtraData: ExtraDataTypes>: _ViewController,
             channelController.sendKeystrokeEvent()
         }
 
-        if content.isEditing {
-            composerView.titleLabel.text = L10n.Composer.Title.edit
+        switch content.state {
+        case .new:
+            composerView.inputMessageView.textView.placeholderLabel.text = L10n.Composer.Placeholder.message
             Animate {
-                self.composerView.confirmButton.isHidden = false
-                self.composerView.sendButton.isHidden = true
-                self.composerView.headerView.isHidden = false
+                self.composerView.confirmButton.isHidden = true
+                self.composerView.sendButton.isHidden = false
+                self.composerView.headerView.isHidden = true
             }
-        } else if content.isQuoting {
+        case .quote:
             composerView.titleLabel.text = L10n.Composer.Title.reply
             Animate {
                 self.composerView.headerView.isHidden = false
             }
-        } else {
-            composerView.inputMessageView.textView.placeholderLabel.text = L10n.Composer.Placeholder.message
+        case .edit:
+            composerView.titleLabel.text = L10n.Composer.Title.edit
             Animate {
-                self.composerView.confirmButton.isHidden = true
-                self.composerView.sendButton.isHidden = false
-                self.composerView.headerView.isHidden = true
+                self.composerView.confirmButton.isHidden = false
+                self.composerView.sendButton.isHidden = true
+                self.composerView.headerView.isHidden = false
             }
+        default:
+            break
```

Comment:
Maybe we can add some `log.debug` here for debugging purposes? Even though it's not an error, it could be handy to know about this.

Reply:
Yes makes sense, will log here 👍